### ISSUE TITLE
Is this project no loger active?

### ISSUE THREAD

**Issue by JigarPrajpati** [ElasticLINQ/ElasticLINQ#133]
i would like to use this in production, can i use it, or this project are end of life?

**Comment by damieng** [comment]
The published NuGet version is very stable for Elastic 2.x but beyond that Elastic kept changing the query DSL and as I no longer used Elastic in my work it has not kept up.

I would not recommend using it for later versions of Elastic.

**Comment by sllabresrfa** [comment]
I've started using it for our business.  If this is an issue let me know and I'll take it down but I've done some butchering and put a prerelease here (yesterday):

https://dev.azure.com/RedFlagAlert/ElasticLinq/_packaging?_a=feed&feed=ElasticLinq

Apologies as some of my hacking is grim :S  I haven't yet added the UnitTests back in.  I spent some time on this yesterday but the UnitTest project generated over 700 errors as a .NET 5 project.

The butchering I've done adds geo_distance and nested query types.  Understanding IQueryable has been a bit of a mind bender :S

My primary drive is using this library in conjunection with Hot Chocolate GraphQl.  It's working well for us against ElasticSearch 7.15.

Again, if I've done anything inappropriate regarding usage and license let me know and I'll resolve.
